packages/smooth_app/lib/generic_lib/widgets/smooth_back_button.dart

```diff
@@ -25,7 +25,10 @@ class SmoothBackButton extends StatelessWidget {
             padding: _iconPadding,
             child: Icon(
               ConstantIcons.instance.getBackIcon(),
-              color: iconColor ?? Colors.white,
+              color: iconColor ??
+                  (Theme.of(context).colorScheme.brightness == Brightness.light
+                      ? Colors.white
+                      : Colors.black),
```

Comment:
Please remove this part: it has nothing to do with the issue and I believe there's another pending issue/PR on the same subject.
Let's avoid a collision.